Apps/Admin/Client/Pages/SupportPage.razor.cs

```diff
@@ -151,6 +155,13 @@ private IList<MessagingVerificationModel> GetMessagingVerificationModels(string
             return this.MessagingVerifications.Where(v => v.UserProfileId == hdid).ToList();
         }
 
+        private TimeZoneInfo GetTimeZone()
+        {
+            string unixTzValue = this.ConfigurationState.Value.Result?.TimeZone[DateFormatter.UnixTzIdKey] ?? DateFormatter.DefaultUnixTzValue;
+            string windowsTzValue = this.ConfigurationState.Value.Result?.TimeZone[DateFormatter.WindowsTzIdKey] ?? DateFormatter.DefaultWindowsTzValue;
+            return DateFormatter.GetLocalTimeZone(unixTzValue, windowsTzValue);
```

Comment:
It might be easier to update Admin.Client/wwwroot/appsettings.json to add the same config settings that exist on the back-end.  Then you'd just need to inject IConfiguration in this class and could pass that along to the existing GetLocalTimeZone method in DateFormatter.

Reply:
Changed.